examples/iam/serviceaccount/serviceaccount.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: iam.gcp.crossplane.io/v1beta1
+kind: ServiceAccount
+metadata:
+  name: perfect-test-sa
+spec:
+  properties:
+  forProvider:
+    accountId: perfect-account
```

Comment:
I guess updating this yaml is forgotten.

Reply:
updated, thanks :)